troi/playlist.py

```diff
@@ -32,7 +32,7 @@ def read(self, inputs, debug=False):
             listens.append({
```

Comment:
there's a potential error on the line above this (I can't annotate it in this pr), where you call `e.artist.mbids`, but `e.artist` could be None